### PR TITLE
add static class

### PR DESCRIPTION
**Pull request by ankitdas13**

### CODE REVIEW

File: src/main/java/com/razorpay/ApiClient.java [hunk 1]
```diff
@@ -113,6 +129,14 @@ public void delete(String path, JSONObject requestObject) throws RazorpayExcepti
     }
 
     if (statusCode >= STATUS_OK && statusCode < STATUS_MULTIPLE_CHOICE) {
+    	
+      if(!responseJson.has(ENTITY)) {
+    	   String cls = StaticEntity(response);
```
Comment (ersinghlovepreet) [inline src/main/java/com/razorpay/ApiClient.java:134]: StaticEntity method should be populateEntiryFromURL(response.request().url().pathSegments())

File: src/main/java/com/razorpay/StaticClass.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.razorpay;
+
+public class StaticClass {
```
Comment (ersinghlovepreet) [inline src/main/java/com/razorpay/StaticClass.java:3]: This should be enum with URL name mapping

File: src/main/java/com/razorpay/ApiClient.java [hunk 3]
```diff
@@ -113,6 +124,12 @@ public void delete(String path, JSONObject requestObject) throws RazorpayExcepti
     }
 
     if (statusCode >= STATUS_OK && statusCode < STATUS_MULTIPLE_CHOICE) {
+      
+      if(!responseJson.has(ENTITY)) {
+    	  String entityName = getEntityNameFromURL(response.request().url());
+          responseJson.put("entity",entityName);
```
Comment (ersinghlovepreet) [inline src/main/java/com/razorpay/ApiClient.java:130]: "entity" -> ENTITY

**Review by ersinghlovepreet — CHANGES_REQUESTED** [review]

File: src/main/java/com/razorpay/ApiClient.java (hunk 3 above)
Comment (ankitdas13) [reply src/main/java/com/razorpay/ApiClient.java]: i have changed this to `Entity`

**Review by ersinghlovepreet — APPROVED** [review]

**Pull request merged**